test/types/hooks.test-d.ts

```diff
@@ -13,42 +13,46 @@ type TestPayloadType = {
   bar: number
 }
 
+class FastifyErrorImpl extends Error implements FastifyError {
+  public code: string = "FST_ERR_HTTP2_INVALID_VERSION";
```

Comment:
Why this was set?

Reply:
This stands for tests reasons otherwise it will cause reasons cose `code` is not implemented in `FastifyErrorImpl`